api/config/shared/global.proto

```diff
@@ -189,6 +198,36 @@ message Backups {
 			google.protobuf.StringValue root_cert = 1;
 		}
 	}
+
+	message GCS {
+		GCPCredentials credentials = 1;
+		Elasticsearch es = 2;
+		Bucket bucket = 3;
+
+		message GCPCredentials {
+			google.protobuf.StringValue json = 1;
+		}
+
+		message Elasticsearch {
+			// Backup repo settings
+			google.protobuf.BoolValue compress = 2;
+			 google.protobuf.BoolValue application_name = 3;
+			// Snapshot settings
+			google.protobuf.StringValue max_snapshot_bytes_per_sec = 4;
+			google.protobuf.StringValue max_restore_bytes_per_sec = 5;
+			google.protobuf.StringValue chunk_size = 6;
+			// GCS client settings
+			google.protobuf.StringValue read_timeout = 7;
+			google.protobuf.BoolValue connect_timeout = 8;
+		}
+
+		message Bucket {
+			google.protobuf.StringValue project_id = 1;
```

Comment:
I think we dont need `project_id`?

Reply:
I leave it just in case the user wants to specify a project other than the one in the json object. Where can I verify what is used when backups are made? we may want to set an override for this.

Reply:
the problem with leaving it is i don't think its wired up. It doesn't look like ES supports it, though that doesn't matter since we go through an s3 gateway. So probably possible to make it work.
The tests just pass the libraries the json credentials which has it in there. To make it work, I think https://github.com/chef/automate/pull/3609/files#diff-5c03eaadb5db8db333ac4aa908b132ceR529-R540 would need to be taught to do something with ProjectID, verify backup gateway works (looks like everything is wired up), and make sure it can be passed through the restore command

Reply:
Agreed to just removed projectid, will open seperate PR if needed in future